.github/workflows/_build_artifacts.yml

```diff
@@ -281,6 +281,7 @@ jobs:
           # We only version client and gateway
           tags: |
             type=raw,value=latest
+            type=raw,value={{branch}}
```

Comment:
just double-checking you weren't missing a `$` character here?

Reply:
Nope, this is syntax from the docker action.